### ISSUE TITLE
Cloud Build pipeline : delete-default-gateway-routes.sh: not found

### ISSUE THREAD

**Issue by komasoftware** [terraform-google-modules/terraform-google-network#119]
I am running terraform init, plan, apply, destroy for every release of a in-house module using GCP Cloud Build as you can see from below cloudbuild.yaml file.
This was working fine until I started running into below issue where **the default gateway deletion script is not found** when running **terraform apply**. This only happens when running on Cloud Build, I cannot reproduce the problem on my local machine.

Any idea ?

 **/bin/sh: .terraform/modules/xpn/terraform-google-modules-terraform-google-network-e273b5e/scripts/delete-default-gateway-routes.sh: not found**

```
- steps
  # download service account key 
  - name: gcr.io/cloud-builders/gsutil
    args: ['cp', 'gs://XXXXXXXXXX/secrets.json', 'secrets.json']
  # setup test environment
  - name: bash
    args: ["bash", "-c", "mv backend.tf.test backend.tf"]
  # init test environment
  - name: hashicorp/terraform:light
    args: ["init"]
    env:
      - "GOOGLE_APPLICATION_CREDENTIALS=secrets.json"
  # plan test environment
  - name: hashicorp/terraform:light
    args: ["plan", "-input=false", "-var-file=variables.test.tfvars", "-refresh=true", "-out=tf-modules-environment.test.tfplan"]
    env:
      - "GOOGLE_APPLICATION_CREDENTIALS=secrets.json"
      - "CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=secrets.json"
  # apply test environment
  - name: hashicorp/terraform:light
    args: ["apply", "-auto-approve", "tf-modules-environment.test.tfplan"]
    env:
      - "GOOGLE_APPLICATION_CREDENTIALS=secrets.json"
      - "CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=secrets.json"
  - name: hashicorp/terraform:light
    args: ["destroy",  "-auto-approve", "-var-file=variables.test.tfvars"]
    env:
      - "GOOGLE_APPLICATION_CREDENTIALS=secrets.json"
      - "CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=secrets.json"

```
Error: Error running command '.terraform/modules/xpn/terraform-google-modules-terraform-google-network-e273b5e/scripts/delete-default-gateway-routes.sh xpn-terratest-04 terratest-xpn': exit status 127. Output: /bin/sh: .terraform/modules/xpn/terraform-google-modules-terraform-google-network-e273b5e/scripts/delete-default-gateway-routes.sh: not found

![Selection_999(476)](https://user-images.githubusercontent.com/482152/69734095-b620eb80-112e-11ea-89c6-2b617470d901.jpg)

**Comment by morgante** [comment]
Can you share your Terraform configuration and versions?

**Comment by komasoftware** [comment]
**terraform-google-network 1.4.3**

```
module "xpn" {
  source                                 = "terraform-google-modules/network/google"
  version                                = "~> 1.4.3"
  project_id                             = module.xpn_host.project_id
  network_name                           = local.xpn_network_name
  auto_create_subnetworks                = false
  delete_default_internet_gateway_routes = true
  shared_vpc_host                        = true

  subnets          = var.subnets
  secondary_ranges = var.secondary_ranges
}

```
**Terraform v0.12.16** is being pulled in during Cloud Build, same as I run locally.

![Selection_999(477)](https://user-images.githubusercontent.com/482152/69753779-a3211200-1154-11ea-9f09-6564be18e645.jpg)

**Comment by morgante** [comment]
I'm not quite sure what could be causing this, but adding [abspath](https://www.terraform.io/docs/configuration/functions/abspath.html) to the [script call](https://github.com/terraform-google-modules/terraform-google-network/blob/master/main.tf#L100) might help. Could you try that and see if it works?

**Comment by komasoftware** [comment]
I have added a 'bash' build step to verify whether the executable is at the expected path .terraform/modules/xpn/terraform-google-modules-terraform-google-network-e273b5e/scripts/delete-default-gateway-routes
Next I used a buidl step to modify the 'workdir' property for the local-exec directive to point to the correct location.
      name = "bash"
      args = ["-c", "grep -r -l delete-default-gateway-routes .terraform/ | xargs sed -i '/delete-default-gateway-routes/a working_dir=\"\\/workspace\"';  grep -r -l delete-default-gateway-routes .terraform/ | xargs cat"]

None of this resolved the issue. I'll be digging in a bit more.
